mapbox/compat.py

```diff
@@ -0,0 +1,10 @@
+# Python 2/3 compatibility
+
+import sys
+
+if sys.version_info >= (3, ):
+    from urllib.parse import urlparse
+    text_types = (str, )
+else:
+    from urlparse import urlparse
+    text_types = (str, unicode)
```

Comment:
Do we need this file yet? These names don't appear to used anywhere (though imported in uploads.py)